util/zenfs.cc

```diff
@@ -604,6 +637,8 @@ int main(int argc, char **argv) {
     return ROCKSDB_NAMESPACE::zenfs_tool_dump();
   } else if (subcmd == "fs-info") {
     return ROCKSDB_NAMESPACE::zenfs_tool_fsinfo();
+  } else if (subcmd == "force-rmdir") {
```

Comment:
Lets introduce a normal rmdir that would remove a directory if empty, and if it is not empty require the --force flag

Reply:
Good point! This feature is now implemented :)